incubator/patroni/values.yaml

```diff
@@ -25,7 +25,15 @@ Credentials:
   Admin: cola
   Standby: pinacolada
 
-# Patroni depends on etcd, configure it here
+## Distribution Configuration stores. Please note that only one of the following stores should be selected.
 Etcd:
-  Host: # fill-in value for etcd host (etcd.default.svc.cluster.local), leave blank to use the discovery parameter
+  Enable: true
```

Comment:
Should these follow the recommendations under - https://github.com/kubernetes/helm/blob/9cfdbd1f7773aff6d3de0080796fc10559b91948/docs/chart_best_practices/requirements.md#conditions-and-tags